Source/DafnyRuntime/dafnyRuntime.dfy

```diff
@@ -0,0 +1,23 @@
+method HasTuples() {
+  var has0 := (0);
```

Comment:
```suggestion
  var has0 := ();
```

The generated files look right anyway as I believe `()` gets declared implicitly anyway.